### PR TITLE
Support elixir

### PR DESCRIPTION
**Pull request by sivsushruth**

### CODE REVIEW

**Comment by benoitc** [comment]
@sivsushruth seems good for me. Can you add some unit tests for it ?

**Comment by sivsushruth** [comment]
@benoitc Adding them now.

File: src/hooks.erl [hunk 1]
```diff
@@ -514,9 +514,9 @@ check_hooks([{_Priority, MFA}=Hook | Rest], Hooks) ->
 
 check_mfa({Mod, Fun, Arity}) ->
   _ = code:ensure_loaded(Mod),
-  case erlang:function_exported(Mod, Fun, Arity) of
-    true -> ok;
-    false -> {error, hooks_not_exported}
+  case lists:member({Fun, Arity}, Mod:module_info(exports)) of
```
Comment (benoitc) [inline src/hooks.erl:517]: why this change?
